pkg/cli/admin/release/extract_tools.go

```diff
@@ -1236,20 +1329,20 @@ func findClusterIncludeConfig(ctx context.Context, restConfig *rest.Config) (man
 		return config, err
 	} else {
 		config.Overrides = clusterVersion.Spec.Overrides
-		config.Capabilities = &clusterVersion.Status.Capabilities
-
-		// FIXME: eventually pull in GetImplicitlyEnabledCapabilities from https://github.com/openshift/cluster-version-operator/blob/86e24d66119a73f50282b66a8d6f2e3518aa0e15/pkg/payload/payload.go#L237-L240 for cases where a minor update would implicitly enable some additional capabilities.  For now, 4.13 to 4.14 will always enable MachineAPI, ImageRegistry, etc..
-		currentVersion := clusterVersion.Status.Desired.Version
-		matches := regexp.MustCompile(`^(\d+[.]\d+)[.].*`).FindStringSubmatch(currentVersion)
-		if len(matches) < 2 {
-			return config, fmt.Errorf("failed to parse major.minor version from ClusterVersion status.desired.version %q", currentVersion)
-		} else if matches[1] == "4.13" {
-			build := configv1.ClusterVersionCapability("Build")
-			deploymentConfig := configv1.ClusterVersionCapability("DeploymentConfig")
-			imageRegistry := configv1.ClusterVersionCapability("ImageRegistry")
-			config.Capabilities.EnabledCapabilities = append(config.Capabilities.EnabledCapabilities, configv1.ClusterVersionCapabilityMachineAPI, build, deploymentConfig, imageRegistry)
-			config.Capabilities.KnownCapabilities = append(config.Capabilities.KnownCapabilities, configv1.ClusterVersionCapabilityMachineAPI, build, deploymentConfig, imageRegistry)
+		// known and baseline may grow from the current cluster version to the oc version
+		capSet := configv1.ClusterVersionCapabilitySetCurrent
+		if capabilitiesSpec := clusterVersion.Spec.Capabilities; capabilitiesSpec != nil &&
+			len(capabilitiesSpec.BaselineCapabilitySet) > 0 {
+			capSet = capabilitiesSpec.BaselineCapabilitySet
+		}
+		deepCopy := clusterVersion.Status.Capabilities.DeepCopy()
+		if capSet == configv1.ClusterVersionCapabilitySetCurrent {
```

Comment:
`v4.y` could be grow too over time.
But it keeps the consistency with 
https://github.com/openshift/oc/blob/def90a65fcbad5d7e54add49975e9c3daa92a4f9/pkg/cli/admin/release/extract_tools.go#L1300-L1302